tools/fastlane-plugin/spec/bugsnag_upload_dsym_action_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 
 Action = Fastlane::Actions::UploadSymbolsToBugsnagAction
+BUGSNAG_CLI_PATH = Action::bundled_bugsnag_cli_path
```

Comment:
Can we add a few tests for using a system CLI and returning a few version ranges?